tests/test_player.py

```diff
@@ -2,27 +2,24 @@
 import unittest
 from game.player import Player
 from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
+from mock import MagicMock
 
 class TestPlayer(unittest.TestCase):
     def setUp(self):
-        self.player_blue = Player('blue')
-        self.player_red = Player('red')
+        strategy_dummy = StrategyOne()
+        strategy_dummy.return_column = MagicMock(return_value = 1)
```

Comment:
As a note: mocks actually allows you to do much interesting things, but are out of the scope of what we are covering now.